src/Orleans/Configuration/GlobalConfiguration.cs

```diff
@@ -81,6 +81,29 @@ public enum ReminderServiceProviderType
         }
 
         /// <summary>
+        /// Configuration for Gossip Channels
+        /// </summary>
+        public enum GossipChannelType
+        {
+            /// <summary>Default value to allow discrimination of override values.</summary>
+            NotSpecified,
```

Comment:
`None = 0` is the conventional enum value for default "uninitialized" values. Would it not fit here?